### ISSUE TITLE
Add frontend for coala.io/links

### ISSUE THREAD

**Issue by supergr35** [coala/landing-frontend#168]
Imo coala.io/links is really useful because it can show newcomers all the easy ways to get to things like newcomer issues or commit docs. However, it doesn't have a graphical frontend. We also might want to put changes in our docs to mention coala.io/links if this is implemented.

**Comment by hemangsk** [comment]
sounds great

**Comment by yukiisbored** [comment]
Yeah, I'm looking for solutions and a better way to handle shortlinks. We're currently using nginx to do that.

**Comment by supergr35** [comment]
I'd kinda like to do this, but it'd most definitely take me a while. Is that alright?

**Comment by supergr35** [comment]
I discussed this with @Mixih and he felt like this could maybe just go in the docs. Anybody's thoughts for that? If it isn't in the docs, then where should we make coala.io/links go?
